Kitodo-DataFormat/src/main/resources/xsd/mets.xsd

```diff
@@ -0,0 +1,1768 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is it possible to retrieve this file from internet on usage instead of putting it into repo and maintain it?